fastai/data/core.py

```diff
@@ -31,7 +31,12 @@ def show_batch(
     return ctxs
 
 # %% ../../nbs/03_data.core.ipynb 10
-@typedispatch
+def get_show_batch_func(x_typ=Any, y_typ=Any, samples_typ=Any):
```

Comment:
@jph00 is this what you had in mind when you said 'make a show_batch_basic' func?
I couln't do exactly that because sometimes the x_typ!=object.

Reply:
How about then adding `show_batch_basic(...)` which calls `get_show_batch_func(object)(...)`? And maybe rename `get_show_batch_func` to `get_show_batch`?